docker/supabase/docker-compose.yml

```diff
@@ -0,0 +1,158 @@
+# Usage
+#   Start:          docker-compose up
+#   With helpers:   docker-compose -f docker-compose.yml -f ./dev/docker-compose.dev.yml up
+#   Stop:           docker-compose down
+#   Destroy:        docker-compose -f docker-compose.yml -f ./dev/docker-compose.dev.yml down -v --remove-orphans
+
+version: '3.8'
+
+services:
+  studio:
+    container_name: supabase-studio
+    image: supabase/studio:latest
+    restart: unless-stopped
+    ports:
+      - ${STUDIO_PORT}:3000/tcp
+    environment:
+      SUPABASE_URL: http://kong:8000
```

Comment:
Above we call this `SUPABASE_API_URL` and they call it `SUPABASE_URL`.  Should we change to use `SUPABASE_URL` everywhere, so we don't confuse things, or misconfigure by having them be different?

Reply:
Sounds good